spatialmath/geometry_test.go

```diff
@@ -37,6 +37,8 @@ func TestGeometrySerialization(t *testing.T) {
 		{"point", GeometryConfig{Type: "point", TranslationOffset: translation, OrientationOffset: orientation, Label: "point"}, true},
 		{"infer point", GeometryConfig{}, false},
 		{"bad type", GeometryConfig{Type: "bad"}, false},
+		{"c", GeometryConfig{Type: "capsule", L: 4, R: 1, TranslationOffset: translation, OrientationOffset: orientation, Label: "c"}, true},
+		{"infer c", GeometryConfig{L: 4, R: 1, TranslationOffset: translation, OrientationOffset: orientation, Label: "infer cc"}, true},
```

Comment:
Label here causing a problem with the x64 tests? Seems odd. Check out test output from the last run for clarity.

Reply:
That's a typo and is my fault, will fix.